wls-gui-wahllokalsystem/src/views/ExampleBackendCommunicationView.vue

```diff
@@ -43,40 +41,31 @@ import {
   VTextField,
 } from "vuetify/components";
 
-import {
-  broadcastMessageRead,
-  getBroadcastMessage,
-  postBroadcastMessage,
-} from "@/api/wls-clients/broadcast-service/broadcast-client";
+import { useBroadcastService } from "@/composables/wlsClients/broadcastService/broadcastMessage";
 
-const messageInput = ref("Broadcast Message");
-const message = ref("");
-const errors = ref({ get: "", post: "", read: "" });
-let messageId = "";
+const { getMessage, postMessage } = useBroadcastService();
 
-async function getMessage(wahlbezirkID: string) {
-  errors.value.get = "";
-  message.value = "";
-  try {
-    const response = await getBroadcastMessage(wahlbezirkID);
-    const content: BroadcastMessageToRead = await response.json();
-    message.value = content.nachricht;
-    messageId = content.oid;
+const messageInput = ref("I am a message");
+const messageToShow = ref("");
+const errorToShow = ref("");
 
-    await broadcastMessageRead(messageId).catch(() => {
-      errors.value.read =
-        "Es ist ein Fehler beim Lesen der Nachricht aufgetreten";
-    });
-  } catch (e) {
-    errors.value.get = (e as Error).message;
+async function getBroadcastMessage(id: string) {
+  errorToShow.value = "";
+  messageToShow.value = "";
+  const { message, error } = await getMessage(id);
+  if (error) {
+    errorToShow.value = error;
   }
```

Comment:
Im Template ist ein v-if dass dafür sorgt dass kein Leerstring, null, oder was anderes dass falsy ist angezeigt wird
```suggestion
  errorToShow.value = error;
```

Reply:
ich wusste nicht, dass das automatisch mit getestet wird 👍🏽 danke

siehe [df08c18](https://github.com/it-at-m/Wahllokalsystem/pull/681/commits/df08c180e4031cb3a7dd86f50daa845112eabcb6)